.circleci/tests/default_values_charts_test.go

```diff
@@ -0,0 +1,28 @@
+package tests
+
+import (
+	"path"
+	"path/filepath"
+)
+
+func (s *helmRepoSuite) TestChartWithDefaultValues() {
```

Comment:
I don't think we need this test, let's just add default.yaml per chart to `TestChartWithCustomValues` and create single test. I prefer it will be opt-in feature.